atlp-rwanda/champs-ec-be
# #187300267 User should be prompted to change their passwords every x amount of time

## What does this PR do?
This PR addresses the requirements outlined in Pivotal ID #187300234.

## Description of Task to be completed?
I was given the task of writing a functionality that checks if the user's password has expired within a given period of time and then prompt that tells him/her to update his/her password when that time of expiration reaches. 

## How should this be manually tested?
Currently how you can test this functionality is by using swagger documentation by trying to signup, verify your email so that you can signin after signing in you can wait for 2 minutes and then try to login, this will fail due to expiration of the password, now update your password and there you go, your password has been updated and now you can login. 

What are the relevant pivotal tracker/Trello stories?
Pivotal ID #187300234.
